components/director/internal/repo/common.go

```diff
@@ -6,18 +6,65 @@ import (
 	"strings"
 )
 
+type ConditionOp string
+
+const (
+	EqualOp    ConditionOp = "="
+	NotEqualOp ConditionOp = "!="
+	IsNull     ConditionOp = "IS NULL"
+	IsNotNull  ConditionOp = "IS NOT NULL"
+)
+
 type Conditions []Condition
 type Condition struct {
 	Field string
+	Op    ConditionOp
 	Val   string
 }
 
+func NewEqualCondition(field, val string) Condition {
+	return Condition{
+		Field: field,
+		Val:   val,
+		Op:    EqualOp,
+	}
+}
+
+func NewNotEqualCondition(field, val string) Condition {
```

Comment:
this method seems to not be used, even in tests: `NewNotEqualCondition` and `NewNullCondition`

Reply:
removed